cluster/gce/util.sh

```diff
@@ -331,6 +331,26 @@ function wait-for-jobs {
   fi
 }
 
+# Robustly try to create a static ip.
+# $1: The name of the ip to create
+function create-static-ip {
```

Comment:
nit: This is going to retry in rapid succession in a tight loop.  Please insert an exponential backoff (even as simple as a "sleep ${attempt * 5}".